modules/gitops-argo-cd-installation.adoc

```diff
@@ -23,4 +23,16 @@ To manage cluster configurations or deploy applications, you can install and dep
 
 .. Create an external OS Route to access Argo CD server. Click *Server* -> *Route* and check *Enabled*.  
 
-. To open the Argo CD web UI, click the route by navigating to *Networking* -> *Routes* -> *<instance name>-server* in the project where the Argo CD instance is installed.
\ No newline at end of file
+. To open the Argo CD web UI, click the route by navigating to *Networking* -> *Routes* -> *<instance name>-server* in the project where the Argo CD instance is installed.
++
+[NOTE]
+====
+To be a user of the `cluster-admins` group, use the `oc adm groups new cluster-admins <user>` command, where `<user>` is the default cluster role that you can bind to users and groups cluster-wide or locally. 
```

Comment:
Fixed.

Reply:
Hello @Dhruv-Soni11 thank you for adding the missing context! Could you please also address the part I mention in the comment above about the `<user>` parameter? Thank you!

**Copy of the comment:**

This note confuses me a little bit.
Because we explain: `<user> is the default role that you can bind to users and groups...`

We explain this in the OCP docs: 
[oc adm groups new](https://docs.openshift.com/container-platform/4.14/cli_reference/openshift_cli/administrator-cli-commands.html#oc-adm-groups-new)
It explains that <user> variable adds a user to the group, not the role.

Am I missing some understanding here? Because as I understand it, "user" and "role" have different meanings.

Reply:
@eromanova97 Yes, please refer to this slack thread: https://redhat-internal.slack.com/archives/C048WUV584F/p1709305166148199?thread_ts=1709277793.271369&cid=C048WUV584F and read till the end for a complete understanding.  So IMO, we can retain the sentence as is. Thank you!

Reply:
Thank you @Dhruv-Soni11 for looking into this and thank you @Srivaralakshmi for your insight! Let's keep the note as it is :smile_cat: